=== RUN   TestA07_ConcurrentRecv
panic: test timed out after 1m0s
	running tests:
		TestA07_ConcurrentRecv (1m0s)

goroutine 33 [running]:
testing.(*M).startAlarm.func1()
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2484 +0x394
created by time.goFunc
	/home/ilian/Tools/go-runtime/src/time/sleep.go:215 +0x2d

goroutine 1 [chan receive]:
testing.(*T).Run(0xc0001a01c0, {0x57ccbb?, 0xc00018cb30?}, 0x585d20)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1859 +0x431
testing.runTests.func1(0xc0001a01c0)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2279 +0x37
testing.tRunner(0xc0001a01c0, 0xc00018cc70)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
testing.runTests(0xc0001ce000, {0x6ab700, 0x1e, 0x1e}, {0x6b1020?, 0x7?, 0x6afea0?})
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2277 +0x4b4
testing.(*M).Run(0xc000190140)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:2142 +0x64a
main.main()
	_testmain.go:103 +0x9b

goroutine 18 [chan receive]:
scenarios.TestA07_ConcurrentRecv(0xc0001a0380?)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go:53 +0x7e
testing.tRunner(0xc0001a0380, 0x585d20)
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1792 +0xf4
created by testing.(*T).Run in goroutine 1
	/home/ilian/Tools/go-runtime/src/testing/testing.go:1851 +0x413

goroutine 19 [chan receive]:
scenarios.TestA07_ConcurrentRecv.func1()
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go:52 +0x25
created by scenarios.TestA07_ConcurrentRecv in goroutine 18
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go:52 +0x66
FAIL	scenarios	60.065s
FAIL
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go
TestName:  TestA07_ConcurrentRecv
Import added at line: 3
Header added at line: 50
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Own/Scenarios/scenarios_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestA07_ConcurrentRecv

Exit Replay with code  10 Timeout
Exit Message: panic: Timeout

FAIL	scenarios	60.164s
FAIL
Bugreport info: 0_1,fail
Bugreport info: 0_2,fail
